python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -665,16 +675,20 @@ def _evaluate_sensor(
         instigator_data.last_tick_timestamp if instigator_data else None,
         instigator_data.last_run_key if instigator_data else None,
         instigator_data.cursor if instigator_data else None,
+        context.log_key,
         instigator_data.last_sensor_start_timestamp if instigator_data else None,
     )
 
     yield
 
+    # Kept for backwards compatibility with sensor log keys that were previously created in the
```

Comment:
I believe this is the only backcompat concern we have to worry about (i.e. the gRPC server is not on the Dagster version where the instigation logger uses the log key created by the sensor daemon).